app/models/occupation_standard.rb

```diff
@@ -250,11 +250,11 @@ def related_job_titles
   end
 
   def source_file
-    data_import.source_file
+    data_import&.source_file
   end
 
   def public_document?
-    source_file.public_document || standards_import.public_document
+    !!(source_file&.public_document || standards_import&.public_document)
```

Comment:
I don't love `!!`. Can we do something more explicit like:

```
doc = source_file&.public_document || standards_import&.public_document
doc.present?
```

Reply:
@littleforest updated